lib/LANraragi/Model/Tankoubon.pm

```diff
@@ -0,0 +1,270 @@
+package LANraragi::Model::Tankoubon;
+
+use feature qw(signatures);
+no warnings 'experimental::signatures';
+
+use experimental "try";
+
+use strict;
+use warnings;
+use utf8;
+
+use Redis;
+use Mojo::JSON qw(decode_json encode_json);
+use List::Util qw(min);
+
+use LANraragi::Utils::Database qw(redis_encode redis_decode invalidate_cache get_archive_json_multi);
+use LANraragi::Utils::Logging qw(get_logger);
+
+# get_tankoubon_list(page)
+#   Returns a list of all the Tankoubon objects.
+# NEEDS PAGINATION
```

Comment:
Outdated comment since this sub now does have pagination?  

Speaking of pagination, since there's no real CPU gain here from adding it (we're still getting the entire list of tanks, just splitting it after the fact), maybe this sub should just return the full array, and the API sub should be the one to handle pagination?  

(I'd also advocate for maybe being able to pass page=-1 to grab everything w/o pagination which I think we're doing in at least one other API call somewhere in the app 🤔)

Reply:
Well, is not that I added for the CPU gain(Which is very relevant BTW, I think Scan probably has an option to limit the results of the keys, but I'm not sure how to use it), but because is not a good practice to return lists of elements that can growth very large in a response, mostly because the response could one day get large enough to cause a timeout.

Still, I added the page=-1 so it can be tested how it works to have it return everything at once.